LanguageFeatures/Augmentations/augmenting_constructors_A09_t03.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion An incomplete constructor can be completed by adding an
+/// initializer list and/or a body, or by adding a redirection.
+///
+/// @description Checks that constructor with the initializing formals can be
+/// augmented.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=augmentations
+
+import '../../Utils/expect.dart';
+
+class C {
+  int x;
+  C(this.x);
+  C.foo([this.x]);
+  C.bar({this.x});
+  C.baz({required this.x});
+}
+
+augment class C {
+  augment C(int x);
+  augment C.foo([int x = 1]);
+  augment C.bar({int x = 2});
+  augment C.baz({required int x});
+}
+
+enum E {
+  e0(0), e1.foo(), e2.foo(2), e3.bar(), e4.bar(x: 3), e5.baz(x: 4);
+  final int x;
+  const E(this.x);
+  const E.foo([this.x]);
+  const E.bar({this.x});
+  const E.baz({required this.x});
+}
+
+augment enum E {
```

Comment:
I can't see grammar support for omitting the part that declares enumerated values, even in an augmenting declaration.

Reply:
Yes, opening my eyes helped, I can see it now. ;-D